pkg/helm/charts/tests/centralservices/testdata/helmtest/central.test.yaml

```diff
@@ -63,26 +63,9 @@ tests:
     .rolebindings["stackrox-central-psp"] | assertThat(. == null)
     .clusterroles["stackrox-central-psp"] | assertThat(. == null)
 
-- name: "central with OpenShift 3 and enabled SCCs"
-  server:
-    visibleSchemas:
-    - openshift-3.11.0
-    availableSchemas:
-    - openshift-3.11.0
-  values:
-    env:
-      openshift: 3
-    system:
-      createSCCs: true
-  expect: |
-    .securitycontextconstraints["stackrox-central"] | .users[0] | assertThat(contains("system:serviceaccount:stackrox:central"))
-    .roles["use-central-scc"] | assertThat(. == null)
-    .rolebindings["central-use-scc"] | assertThat(. == null)
-    .deployments["central"].spec.template.spec.affinity.nodeAffinity | .preferredDuringSchedulingIgnoredDuringExecution | assertThat(length == 5)
-    verifyNodeAffinities(.deployments["central"])
-    envVars(.deployments.central; "central") | assertThat(has("ROX_OPENSHIFT") == true)
 
-- name: "central with OpenShift 4 and disabled SCCs"
+# FIXME: Add test case for OS4 with stock SCCs and set hostPath, to check for hostmount-anyuid
```

Comment:
This should be rather a TODO with a ticket reference. If we leave this FIXME, then CI-style should complain.

Reply:
That's an excellent point, well spotted! I've updated the comment to also include the follow up ticket in [075c207](https://github.com/stackrox/stackrox/pull/8781/commits/075c207cb07073af6196276801b01308b2f89e7b)